.travis.yml

```diff
@@ -8,3 +8,9 @@ env:
   - STACK=cedar-14
 script:
   - bin/docker-build.sh $STACK
+  - |
```

Comment:
I agree -- originally I was hoping it would be more of a one-liner but with trying to handle both the "dirty working directory" and "new untracked files" case that isn't possible with Git at present. Perhaps this can go in `bin/runtests.sh` and in the future additional tests be added there (though admittedly failing tests if the working directory is dirty does seem weird locally..).

I'll hold off updating this PR until other PRs add whatever missing packages are deemed required (I'm presuming the whole of `libgnome2-0`, `libgnomevfs2-0`, `libgconf2-4`, `systemd` plus deps might be unnecessary?) .